src/users/api/views.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth import get_user_model
+from django.http import JsonResponse
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
+
+from core.authentication import TokenAuthentication
+
+User = get_user_model()
+
+
+@api_view(['GET'])
+@authentication_classes([TokenAuthentication])
+@permission_classes([IsAuthenticated])
+def user_list(request):
+    role = request.GET.get('role')
+    if not role or role != "Reviewer":
+        return JsonResponse({'detail': 'role is not given or invalid.'}, status=400)
+    qs = User.objects.filter(is_active=True,verified=True, groups__name= "Reviewer")
```

Comment:
nit
It's quite common that we need to filter by is_valid and verified.
Consider writing a new queryset method similar to `UserQuerySet.get_valid_speakers()` and use it here